src/components/tile/tile.pw.tsx

```diff
@@ -0,0 +1,263 @@
+      await assertCssValueIsApproximately(dd, "margin-left", 0);
+    });
+  });
+
+  tileFooterVariants.forEach(([variant, backGroundColor]) => {
+    test(`check Tile Footer variant as ${variant}`, async ({ mount, page }) => {
+      await mount(
+        <TileFooterComponent variant={variant} data-element="tile-footer" />
+      );
+
+      const footer = getDataElementByValue(page, "tile-footer");
+      await expect(footer).toHaveCSS("background-color", backGroundColor);
+      await checkCSSOutline(
+        footer,
+        "1px",
+        "border-top",
+        "solid",
+        "rgb(204, 214, 219)"
+      );
+    });
+  });
+
+  characters.forEach((children) => {
+    test(`check Tile Footer children as ${children}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <TileFooterComponent data-element="tile-footer">
+          {children}
+        </TileFooterComponent>
+      );
+
+      const footer = getDataElementByValue(page, "tile-footer");
+      await expect(footer).toHaveText(children);
+    });
+  });
+
+  tileDlWidths.forEach(([w, dtWidth]) => {
+    test(`check width as ${w}`, async ({ mount, page }) => {
+      await mount(<DlTileComponent w={w} />);
+
+      const dt = getDataElementByValue(page, "dt").nth(0);
+      await assertCssValueIsApproximately(dt, "width", dtWidth);
+    });
+  });
+
+  borderVariants.forEach(([borderVariant, borderColor]) => {
+    test(`check border variant as ${borderVariant}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <TileComponent data-element="tile" borderVariant={borderVariant} />
+      );
+
+      const tile = getDataElementByValue(page, "tile");
+      await expect(tile).toHaveCSS("border-color", borderColor);
+    });
+  });
+
+  borderWidths.forEach(([borderWidth, pixelWidth]) => {
+    test(`check border width as ${borderWidth}`, async ({ mount, page }) => {
+      await mount(
+        <TileComponent data-element="tile" borderWidth={borderWidth} />
+      );
+
+      const tile = getDataElementByValue(page, "tile");
+      await expect(tile).toHaveCSS("border-width", pixelWidth);
+    });
+  });
+
+  tileRoundness.forEach((roundness) => {
+    test(`check ${roundness} border radius is as expected`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TileComponent data-element="tile" roundness={roundness} />);
+
+      const result = roundness === "default" ? "8px" : "16px";
+      const tile = getDataElementByValue(page, "tile");
+      await expect(tile).toHaveCSS("border-radius", result);
+    });
+  });
+
+  test("check that any falsy children passed are not rendered", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<TileComponentWithFalsyChildren data-element="tile" />);
+
+    const tileContentCount = await getDataElementByValue(
+      page,
+      "tile-content"
+    ).count();
+    expect(tileContentCount).toEqual(1);
+  });
+});
```

Comment:
Can you add the accessibility tests in please?

Reply:
I've missed some but I'd argue most add no value, what accessibility issue are we testing for when a `Dl` is in a `Tile` with text-align updated for example?

Reply:
Can you wrap the accessibility tests in a `describe` please to group them together? It makes them more readable and easier to run just that group of tests in either VS Code or the Playwright UI if need be.

In terms of the value of these, some may not be needed. They would need analysing on a case by case basis to determine which ones were, and how we can set up a component to check for things that may not be detectable in its default state. Such as making sure the Accordion example is extended first before asserting (which is not included as a test here), otherwise it would bring no more value than the default test of the Accordion itself. We may miss regressions if we make assumptions on what could possibly cause a violation. For instance, not having a test for the `with TileFooter` example containing buttons as it does in Storybook would result in missing the fact that there is currently a violation in that example due to the contrast ratio between the buttons and the background in the footer. This is the only opportunity we have of catching those regressions other than during a manual test which will not be done unless this is the component that changes are being made on.

Reply:
Agreed on slack to wait until we have a standardised approach from QA before making this change